pandas/core/window/rolling.py

```diff
@@ -1051,12 +1019,12 @@ def _get_window(
             # GH #15662. `False` makes symmetric window, rather than periodic.
             return sig.get_window(win_type, window, False).astype(float)
 
-    def _get_weighted_roll_func(
-        self, cfunc: Callable, check_minp: Callable, **kwargs
-    ) -> Callable:
-        def func(arg, window, min_periods=None, closed=None):
-            minp = check_minp(min_periods, len(window))
-            return cfunc(arg, window, minp, **kwargs)
+    @staticmethod
```

Comment:
instead of static can you make this a module level function, or is there a reason you want this as static?